docs/data/product/dea-intertidal/_details.md

```diff
@@ -0,0 +1,308 @@
+## Background
+
+Intertidal environments contain many important ecological habitats such as sandy beaches, tidal flats, rocky shores, and reefs. These environments also provide many valuable benefits such as storm surge protection, carbon storage, and natural resources. 
+
+Intertidal zones are being increasingly faced with threats including coastal erosion, land reclamation (e.g. port construction), and sea level rise. These regions are often highly dynamic, and accurate, up-to-date elevation data describing the changing topography and extent of these environments is needed. However, this data is expensive and challenging to map across the entire intertidal zone of a continent the size of Australia.  
+
+The intertidal zone also forms a critical habitat and foraging ground for migratory shore birds and other species. An improved characterisation of the exposure patterns of these dynamic environments is important to support conservation efforts and to gain a better understanding of migratory species pathways.  
+
+The DEA Intertidal Suite provides annual continental-scale elevation and exposure product layers for Australia’s intertidal zone, mapped at a 10m resolution, from DEA’s archive of open-source Landsat and Sentinel-2 satellite data. These intertidal products enable users to better monitor and understand some of the most dynamic regions of Australia’s coastlines.  
+
+## Applications
+
+* Integration with existing topographic and bathymetric data to seamlessly map the elevation of the coastal zone. 
+
+* Providing baseline elevation data for predicting the impact of coastal hazards such as storm surges, tsunami inundation, or future sea-level rise. 
+
+* Investigating coastal erosion and sediment transport processes. 
+
+* Supporting habitat mapping and modelling for coastal ecosystems extending across the terrestrial to marine boundary. 
+
+* Characterise the spatio-temporal exposure patterns of the intertidal zone to support migratory species studies and applications. 
+
+## What this product offers
+
+The DEA Intertidal product suite is the next generation of intertidal products developed in DEA. It improves on the DEA Intertidal Elevation Model (also known as the National Intertidal Digital Elevation Model or NIDEM)  (Bishop-Taylor et al., 2019) and adds several new features and products to help users better understand the intertidal environment.  
```

Comment:
@benji-glitsos-ga What's the recommended format for linking to other KH articles? Absolute URLs or relative?

Reply:
We use relative links for linking to other KH articles but really, it's a minor point that I don't expect stakeholders to worry about. I'll change these.